### ISSUE TITLE
Feature request: add test release option

### ISSUE THREAD

**Issue by olegsu** [komodorio/helm-dashboard#115]
It would be great also to be able to test the release from the UI instead of running the `helm test ...` command

**Comment by undera** [comment]
Thanks for the suggestion!

**Comment by hdm23061993** [comment]
@undera Can you please assign this to me?

**Comment by ronahk** [comment]
@hdm23061993 Can this be closed?

**Comment by undera** [comment]
I guess the corresponding 1.0 change PR is also needed

**Comment by hdm23061993** [comment]
#178 was created on top of v0.3.1
Need to create a new PR for v1.0.0

The same was finalized on the original PR
> @undera Please help review
> 
> Will port to be compatible with release v1.0.0 once this is merged

**Comment by undera** [comment]
Implemented as part of 1.0
